braindecode/datasets/base.py

```diff
@@ -119,13 +121,30 @@ def __init__(self, windows, description=None, transform=None):
                 'i_stop_in_trial']].to_numpy()
 
     def __getitem__(self, index):
+        """Get a window and its target, or a list of windows and targets.
+
+        Parameters
+        ----------
+        index : int
+            Index to the window (and target) to return.
+
+        Returns
+        -------
+        np.ndarray
+            Window of shape (n_channels, n_times).
+        int | np.ndarray
+            Target(s) for the windows.
+        np.ndarray
+            Crop indices.
+        """
```

Comment:
hm but now this only works for BaseConcatDataset not for BaseDataset right? like there is no code change in function?

Reply:
My bad, this is an artifact of a previous version. I'll clean up the docstring.